helpers/utils.h

```diff
@@ -344,4 +349,40 @@ namespace Utils
     bool isPluginActivated(const char* callSign);
 
     bool getRFCConfig(char* paramName, RFC_ParamData_t& paramOutput);
+
+    struct Telemetry
+    {
+        static void init()
+        {
+#ifdef ENABLE_TELEMETRY_LOGGING
+            t2_init("Thunder_Plugins");
```

Comment:
Pls add a FindTelemetry.cmake under cmake folder to find the telemetry 2.0 header files and libraries. You can then add Find_package to all the places where LOGERR is used.

Reply:
Enabling, linking and compilation dependency is set in another review (part of the verification build) https://gerrit.teamccp.com/#/c/476390/1/recipes-extended/rdkservices/rdkservices_git.bb

Reply:
ok sounds good.